modules/deb.sh

```diff
@@ -221,6 +221,22 @@ function deb_get_and_install_package() {
     deb_install_package "work/deb-packages/${DEB_NAME}" "work/rootfs/"
 }
 
+#  Install binary files of a deb package into work/deb-packages from a local directory
+#  $1 - Directory path
+#
+function deb_get_and_install_input_package() {
+    if ! [[ $# -eq 1 ]]; then
+        log_fatal "deb_get_and_install_input_package() requires 1 argument"
+    fi
+    local package_path="$1"
+    local packages=$(ls ${package_path}/*.deb)
+
+    for package in $packages; do
```

Comment:
This will break if any files contains spaces. See [here](https://github.com/mendersoftware/monitor-client/blob/45ac1a1e4620a5a7cd2096fbd4d5503eedcff9d1/src/daemon.sh#L165) for an example of how we have solved this in other places and [here](https://mywiki.wooledge.org/BashPitfalls#for_f_in_.24.28ls_.2A.mp3.29) for a nice explanation of the issue (I am saving this page in my bookmarks for future me :smile: )